proto/regen/ecocredit/v1/query.proto

```diff
@@ -56,10 +56,10 @@ service Query {
     option (google.api.http).get = "/regen/ecocredit/v1/projects/{project_id}";
   }
 
-  // Batches queries for all batches in the given project with pagination.
+  // Batches queries for all batches with pagination.
   rpc Batches(QueryBatchesRequest) returns (QueryBatchesResponse) {
     option (google.api.http).get =
-        "/regen/ecocredit/v1/projects/{project_id}/batches";
+        "/regen/ecocredit/v1/batches";
```

Comment:
Instead of adding another query for batches by projects, we can make `project_id` optional.